### PR TITLE
Book app

### PR DESCRIPTION
**Pull request by Tracy-miranja**
- [ ] Modify the Awesome books application to have:

- A Navigation bar.
- Three content sections:
- Books list.
- Add book form.
- Contact info

- [ ] Make sure that when you click on any of the links, the main area changes to display only one of the three content sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,10 +8,21 @@
...
+          <li id="contact">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <p id="dateTime"></p>
     <div class="collection" id="book-collection">
       <h2 class="book-heading">Awesome Books</h2>
       <div id="book-list"></div>
-      <div class="sec">
+      <div class="sec" id="box">
```
Comment (muneebulrehman) [inline index.html]: When I reload the app, on the list page, the form to add the books is also visible, which should not be the case. I guess you forgot to set the class `hidden` for the form section by default if you want the list to be the main section to be visible when the app is reloaded. Below is the screenshot of your app when I reload it. Kindly fix this issue.
image

File: index.html [hunk 2]
```diff
@@ -32,6 +43,22 @@ <h3 class="tr">Add a new book</h3>
...
+      </p>
+      <ul>
+        <span>Our email: jumahqueenter@gmail.com</span>
+        <span>Our email: barazatracy44@gmail.com</span>
+        <span>Our phone number: +254710976397</span>
+        <span>Our phone number: +254769212978</span>
+        <span>Our address: Nairobi, Kenya, 5577-00100</span>
+      </ul>
+    </section>
+    <footer class="footer">All Rights are reserved .</footer>
```
Comment (muneebulrehman) [inline index.html:60]: As per the wireframe provided by the microverse, the footer is at the bottom of the page. But in your case, it sticks to its nearest element. Kindly fix the issue.
| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @Tracy-miranja  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**